server/api/api.go

```diff
@@ -17,8 +17,9 @@ type errHandleFunc func(http.ResponseWriter, *http.Request, map[string]string) e
 type API struct {
 	stencilv1beta1.UnimplementedStencilServiceServer
 	grpc_health_v1.UnimplementedHealthServer
-	Namespace domain.NamespaceService
-	Schema    domain.SchemaService
+	Namespace     domain.NamespaceService
+	Schema        domain.SchemaService
+	SearchService domain.SearchService
```

Comment:
@ramey @harikrishnakanchi Should we follow consistent convention here?

Reply:
We also have a method called `Search` which is the implementation of the gRPC service so cannot have a struct field with the same name. What we can do here is probably use embedded structs like
```
type API struct {
        stencilv1beta1.UnimplementedStencilServiceServer
	grpc_health_v1.UnimplementedHealthServer
        domain.NamespaceService
        domain.SchemaService
        domain.SearchService
}
```

Reply:
what convention are we talking about here?

this makes calls from handlers looks like this
```
a.SchemaService.ListVersions()
```

currently it looks like this.
```
a.Schema.ListVersions()
```
I would prefer current one.

regardless of that, I wouldn't touch this api struct as a part of this PR. let's discuss this separately.

Reply:
`SearchService domain.SearchService` ->  `Search domain.SearchService`